package.json

```diff
@@ -9,7 +9,8 @@
     "codegen": "envio codegen",
     "dev": "envio dev",
     "test": "pnpm mocha",
-    "start": "ts-node generated/src/Index.bs.js"
+    "start": "ts-node generated/src/Index.bs.js",
+    "migrate": "ts-node src/migrate.ts"
```

Comment:
I left this script here but it should be deleted, just wanted to save the code in git incase